cmd/cluster/cluster.go

```diff
@@ -52,6 +53,7 @@ func NewCreateCommands() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&opts.ServiceCIDR, "service-cidr", opts.ServiceCIDR, "The CIDR of the service network")
 	cmd.PersistentFlags().StringVar(&opts.PodCIDR, "pod-cidr", opts.PodCIDR, "The CIDR of the pod network")
 	cmd.PersistentFlags().BoolVar(&opts.Wait, "wait", opts.Wait, "If the create command should block until the cluster is up. Requires at least one node.")
+	cmd.Flags().BoolVar(&opts.ExposeKubeAdminPassword, "expose-kube-admin-pwd", opts.ExposeKubeAdminPassword, "Expose kubeadmin password in HostedCluster status")
```

Comment:
can we be consistent with the API field description? Setting status or not seems circumstancial, This is actually dictating the creation or not of an admin user right?